recipe/meta.yaml

```diff
@@ -6,12 +6,12 @@ package:
 
 source:
   fn: setuptools_scm-{{ version }}.tar.gz
-  url: https://github.com/pypa/setuptools_scm/archive/{{ version }}.tar.gz
-  sha256: 9865a6b40e11dc3f4e4020110cd42b9e9cab04be9fc855984fcb4bbdbc8c5d1e
+  url: https://pypi.python.org/packages/80/b7/31b6ae5fcb188e37f7e31abe75f9be90490a5456a72860fa6e643f8a3cbc/setuptools_scm-1.15.0.tar.gz
```

Comment:
Please change

`https://pypi.python.org/packages/80/b7/31b6ae5fcb188e37f7e31abe75f9be90490a5456a72860fa6e643f8a3cbc/setuptools_scm-1.15.0.tar.gz`

to 

`https://pypi.io/packages/source/s/setuptools_scm/setuptools_scm-{{ version }}.tar.gz`